sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ToStringNoNull.CSharp9.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
```

Comment:
what happens if inside a top-level-statement file, you add a local function

`string ToString() { return null; }`?

Reply:
I've added a new test case: it will not raise an issue since it's not a ToString override.